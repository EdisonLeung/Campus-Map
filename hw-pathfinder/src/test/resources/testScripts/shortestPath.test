# Tests pathfinding on a path with the lowest cost and least number of
# jumps

# Create Graph
CreateGraph shortestPath

# Add nodes
AddNode shortestPath a
AddNode shortestPath b
AddNode shortestPath c

# Add edges
AddEdge shortestPath a b 5.0
AddEdge shortestPath b c 5.0
AddEdge shortestPath a c 1.0

# test pathfinding
FindPath shortestPath a c