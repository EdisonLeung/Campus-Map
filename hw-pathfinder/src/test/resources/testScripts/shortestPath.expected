# Tests pathfinding on a path with the lowest cost and least number of
# jumps

# Create Graph
created graph shortestPath

# Add nodes
added node a to shortestPath
added node b to shortestPath
added node c to shortestPath

# Add edges
added edge 5.000 from a to b in shortestPath
added edge 5.000 from b to c in shortestPath
added edge 1.000 from a to c in shortestPath

# test pathfinding
path from a to c:
a to c with weight 1.000
total cost: 1.000